WebApplication/Controllers/AdoptProjectController.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System.Web;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
```

Comment:
Please use native .NET JSON classes... Newtonsoft was not required before this commit

Reply:
Actually we do use Newtonsoft within the project. But I have removed it from my code.